app/src/sharedTest/java/org/oppia/android/app/testing/ImageRegionSelectionInteractionViewTest.kt

```diff
@@ -280,6 +281,68 @@ class ImageRegionSelectionInteractionViewTest {
     }
   }
 
+  @Test
+  // TODO(#1611): Fix ImageRegionSelectionInteractionViewTest
+  @Ignore
```

Comment:
Why are we writing ignored tests? Is it because of #1611 ?

Reply:
Ah this is unfortunate. It seems like Espresso probably is the only choice then.

Reply:
Actually, given this can we use OppiaTestRule and ``RunOn(TestPlatform.ESPRESSO)`` here instead of ignoring? We don't actually need to ignore tests failing for specific platforms anymore.